site/src/components/layout.js

```diff
@@ -202,9 +206,12 @@ const Layout = ({ children, pageContext }) => {
           },
         ]}
       />
+      {!pageSlug.includes('/cookie-settings') && (
+        <CookieModal contentSource={{ ...getCookieConsent(language, setLanguage), focusTargetSelector: headerId }} />
```

Comment:
Accepting cookies does not work in the demo. The cookie modal will not go away, and in browser console there's some piwik errors:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/2777633/183896912-f9af7a10-adde-4cd5-80dd-cb505195d60b.png">